pkgs/tools/misc/yubikey-manager/default.nix

```diff
@@ -2,14 +2,14 @@
 
 python3Packages.buildPythonPackage rec {
   pname = "yubikey-manager";
-  version = "4.0.8";
+  version = "4.0.9";
   format = "pyproject";
 
   src = fetchFromGitHub {
     repo = "yubikey-manager";
-    rev = version;
+    rev = "refs/tags/${version}";
```

Comment:
@ryantm Why is this change being made?

Reply:
https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8